addons/stock/views/stock_quant_views.xml

```diff
@@ -87,17 +87,17 @@
                             <label for="quantity" string="Quantity On Hand"/>
                             <div class="o_row">
                                 <field name="quantity"/>
-                                <field name="product_uom_id" groups="uom.group_uom" options="{'no_open': True}"/>
+                                <field name="product_uom_id" groups="uom.group_uom" widget="many2one_uom" options="{'no_open': True}"/>
                             </div>
                             <label for="available_quantity" string="Available Quantity"/>
                             <div class="o_row">
                                 <field name="available_quantity"/>
-                                <field name="product_uom_id" groups="uom.group_uom" options="{'no_open': True}"/>
+                                <field name="product_uom_id" groups="uom.group_uom" widget="many2one_uom" options="{'no_open': True}"/>
                             </div>
                             <label for="reserved_quantity" string="Quantity Reserved"/>
                             <div class="o_row">
                                 <field name="reserved_quantity"/>
-                                <field name="product_uom_id" groups="uom.group_uom" options="{'no_open': True}"/>
+                                <field name="product_uom_id" groups="uom.group_uom" widget="many2one_uom" options="{'no_open': True}"/>
```

Comment:
Since `product_uom_id` is always `readonly` for quants, this widget doesn't seem useful for any of its views. Should probably be double checked everywhere else this is being added to avoid using it where it's not useful